src/Sulu/Bundle/CategoryBundle/Tests/Functional/Controller/CategoryControllerTest.php

```diff
@@ -173,7 +173,7 @@ public function testGetById()
 
         $client->request(
             'GET',
-            '/api/categories/' . $this->category1->getId()
+            '/api/categories/' . $this->category1->getId() . '?locale=en'
```

Comment:
In the CategoryController the locale is not mandatory. When no locale is passed, the categories are returned in their fallback locale.